chain-impl-mockchain/src/block/mod.rs

```diff
@@ -8,9 +8,9 @@ use std::slice;
 
 mod builder;
 //mod cstruct;
-mod header;
+pub mod header;
 mod headerraw;
-mod version;
+pub mod version;
```

Comment:
this should not be necessary because we expose all the interfaces in the `pub use self::version::*` below

Reply:
Done